cqrs-user-command-service/src/test/java/dev/amir/usercommand/framework/output/sql/mapper/UserJpaMapperTest.java

```diff
@@ -33,8 +30,8 @@ void test_Convert_FromUser_ToUserJpa() {
         UserJpa actual = underTest.convert(expected);
 
         assertNotNull(actual);
-        assertEquals(expected.getRoleId().getValue(), actual.getRoleId());
-        assertEquals(expected.getId().getValue(), actual.getId());
+        assertEquals(expected.getRoleId().getValue().toString(), actual.getRoleId().toString());
```

Comment:
Remove `.getValue()`. it will change `expected.getRoleId().getValue().toString()` to `expected.getRoleId().toString()`